addons/README.md

```diff
@@ -10,6 +10,20 @@ Addons are scripts that analyses Cppcheck dump files to check compatibility with
   Checks Linux system for [year 2038 problem](https://en.wikipedia.org/wiki/Year_2038_problem) safety. This required [modified environment](https://github.com/3adev/y2038). See complete description [here](https://github.com/danmar/cppcheck/blob/main/addons/doc/y2038.txt).
 + [threadsafety.py](https://github.com/danmar/cppcheck/blob/main/addons/threadsafety.py) 
   Analyse Cppcheck dump files to locate threadsafety issues like static local objects used by multiple threads.
++ [naming.py](https://github.com/danmar/cppcheck/blob/main/addons/naming.py)
++ [namingng.py](https://github.com/danmar/cppcheck/blob/main/addons/namingng.py)
+
+### Other files
+
+- cppcheck.py
```

Comment:
What are all these files? Are more of them addons or examples or what? Any additional information would be great. I looked at most of them but couldn't really tell. I also didn't check the history yet.